newsrevenuehub/rev-engine
# [NRH-288] Accepting flagged-recurring causes 500

#### What's this PR do?
Fixes a bug in which accepting a flagged recurring donation from the org dashboard caused a 500. This was being caused by trying to access validated_data on a payment manager instantiated with a contribution instance, not data, as is the case when accepting a flagged recurring donation. 

While we were in there, we:
1. Added a back button to the donation edit page
2. Added an "Frequency" column to the donations list

#### How should this be manually tested?
Go to a live donation page, make a recurring donation using an email that you know will be flagged. Then, logged in as an org admin for the org you just donated to, view your donations in the dashboard. Click on the list item for the donation you just made. Scroll down and click "Accept". It should accept successfully, rather than throwing an error.

Also, notice the back button in the donation page edit view.
And notice the new column "Frequency" in the donations list.


#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
